bricks/template/__brick__/{{project_name.snakeCase()}}/lib/api/response/user_response.dart

```diff
@@ -13,4 +14,9 @@ class UserResponse {
       _$UserResponseFromJson(json);
 
   Map<String, dynamic> toJson() => _$UserResponseToJson(this);
+
+  User toUser() => User(
+        email: email,
+        username: username,
+      );
```

Comment:
This violates the Clean Architecture approach. Response models in the Data layer don't directly connect to models in the Domain layer. We should move this to the `User` class instead as an extension of the `UserResponse` 🤔 

For example https://github.com/luongvo/flutter-survey/blob/8de8375734c323d507aa89ce777d97db3ed87a21/lib/models/user.dart#L25-L32